app/assets/stylesheets/local/site_wide.scss

```diff
@@ -112,14 +112,14 @@ hr.brand {
 // We set 'p' tags within a .branded-body-font -- with some exceptions -- to use our serif font, and have
 // appropriate max-width and spacer for that font.
 .branded-body-font, .scihist-main-layout {
-  p:not(.collection-title):not(.sans-serif), dl:not(.sans-serif) dd, table:not(.sans-serif) td, table:not(.sans-serif) th, .serif {
+  p:not(.featured-topic-title):not(.sans-serif), dl:not(.sans-serif) dd, table:not(.sans-serif) td, table:not(.sans-serif) th, .serif {
     @extend %text-font;
```

Comment:
Hmm, you *removed* `not(".collection-title")` -- but do we need that for another part of the app? Is it safe to remove?

What this is doing is saying that all `p` (paragraphs) will be styled with our "text" styling, _except_ for certain ones -- could we also just make  the relevant collection-title not a `p`?  If it's a title, perhaps it should be a hN tag of some kind? Or just a div?

Reply:
Nah, this is a mistake on my part, but now that I'm looking at it again, I see that I can and should refactor this in such a way that site_wide.scss doesn't need to change. Let me give this another shot. Yeah, we can have this be an h3.

Reply:
OK - now an h3, and the rules that apply to it are stated explicitly in homepage.scss .